io/disk/Avago_storage_adapter/avago9361_vd.py

```diff
@@ -99,16 +108,72 @@ def test_maxvd(self):
                     self.vd_details()
                 self.vd_delete()
 
+    def test_init(self):
+
+        """
+        Test case to start Fast and Full init on VD
+        """
+        self.vd_create('WB', 'ra', 'cached', 256)
+        self.full_init()
+        self.vd_delete()
+
     def test_cc(self):
 
         """
         Function to do consistency operations on VD
         """
         self.vd_create('WT', 'nora', 'direct', 256)
         self.vd_details()
+        self.full_init()
         self.cc_operations()
         self.vd_delete()
 
+    def test_jbod(self):
+
+        """
+        Function to format a drive to JBOD
+        """
+        cmd = "./storcli64 /c%d set jbod=on" % self.controller
+        self.check_pass(cmd, "Failed to set JBOD on")
+        cmd = "./storcli64 /c%d show jbod" % self.controller
+        self.check_pass(cmd, "Failed to show the JBOD status")
+        cmd = "./storcli64 /c%d/eall/sall set jbod" % self.controller
+        self.check_pass(cmd, "Failed to convert the drives to JBOD")
+        cmd = "./storcli64 /c%d show " % self.controller
+        self.check_pass(cmd, "Failed to show the adapter deatils")
+        cmd = "./storcli64 /c0/eall/sall set good force"
+        self.check_pass(cmd, "Failed to set JBOD drives to good")
+
+    def test_online_offline(self):
+
+        """
+        Test to run drive Online/Offline
+        """
+        self.vd_create('WT', 'nora', 'direct', 256)
+        self.vd_details()
+        for _ in range(0, 5):
+            for state in ['offline', 'online']:
+                self.set_online_offline(state)
+                time.sleep(10)
```

Comment:
Reason, I have sleep here is to have a raid array in degraded mode before bringing it online.